students/DiannaTingg/Session03/mailroom.py

```diff
@@ -0,0 +1,124 @@
+# Lesson 03 Assignment: Mailroom, Part 1
+
+# Make a list that holds donor names and a history of the amounts they donated
+donors = [("Jimmy Fallon", [500.00, 700.00]), ("Taylor Swift", [1000.00]), ("Dan White", [200.00, 300.00, 400.00]),
+          ("Trevor Noah", [750.00, 850.00]), ("Elon Musk", [3000.00]), ("Selena Gomez", [100.00, 150.00, 200.00])]
+
+
+# Display menu options
+def display_menu():
```

Comment:
You can remove most or all of the blank rows to make your functions more compact.